src/wasm.rs

```diff
@@ -1380,7 +1384,13 @@ impl WasmModule {
         let mut function_indexes: Vec<u32> = Vec::new();
         loop {
             match p.read() {
-                &ParserState::ElementSectionEntryBody(ref v) => function_indexes.extend(v.iter()),
+                &ParserState::ElementSectionEntryBody(ref v) => {
+                    for elem in v.iter() {
+                        if let ElementItem::Func(idx) = elem {
```

Comment:
What are the other options? Should we at least log here?

Reply:
Added with unimplemented panics.